worker/introspection/script.go

```diff
@@ -251,6 +251,9 @@ juju_revoke_lease () {
   juju_agent --post leases/revoke model="$model" lease="$lease" ns="$ns"
 }
 
+juju_reload_config() {
+  sudo kill -SIGHUP $(pidof jujud)
```

Comment:
`sudo pkill -SIGHUP jujud`?

Regardless, this seems like a bit of a hammer. I guess non machine/controller jujud processes will just ignore SIGHUP?

What about jujud's running in lxd containers?

Reply:
It 100% is a hammer, it's whilst we don't have a charm to test.

> I guess non machine/controller jujud processes will just ignore SIGHUP?

Correct.

> What about jujud's running in lxd containers?

It's fine.